test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -3292,6 +3294,30 @@ TEST_P(LdsTest, HttpFiltersEnabled) {
   gpr_unsetenv("GRPC_XDS_EXPERIMENTAL_FAULT_INJECTION");
 }
 
+// Tests that we ignore filters after the router filter.
+TEST_P(LdsTest, IgnoresHttpFiltersAfterRouterFilter) {
```

Comment:
Just curious, why are we adding this test case now?

Reply:
As I said in the PR description: "Also add a missing test from the fault injection PR, addressing a previously added TODO."